src/getModule/parseInputs/parse.ts

```diff
@@ -1,11 +1,18 @@
+import { PublicClient } from 'viem'
 import { Extras, FormattedAbiParameter } from '../../types'
 import { tuple, tupleArray, stringNumber, decimals, any } from './utils'
+import { DECIMALS_ABI } from '../../getDeploy/constants'
 
-export default function parse(
-  input: FormattedAbiParameter,
-  arg: any,
+export default async function parse(
+  inputs: FormattedAbiParameter[],
```

Comment:
this is a problem parse is the default function of the parseInputs and meant to be used as a recursive function